Tailscale/shared/Tailscale.sh

```diff
@@ -2,6 +2,7 @@
 CONF=/etc/config/qpkg.conf
 QPKG_NAME="Tailscale"
 QPKG_ROOT=`/sbin/getcfg ${QPKG_NAME} Install_Path -f ${CONF}`
+QPKG_PORT=`/sbin/getcfg ${QPKG_NAME} Service_Port -f ${CONF}`
```

Comment:
Service_Port is not defined by the Tailscale package. I don't know what it is defined as. Why do you believe it should be used here?

Reply:
https://github.com/tailscale/tailscale-qpkg/blob/9d60479d10ac2e113953442d8bf21af959ead6c9/Tailscale/qpkg.cfg.in#L31

QPKG_SERVICE_PORT is defined in the file qpkg.cfg.in

and in the file: /etc/config/qpkg.conf

```
[Tailscale]
Name = Tailscale
Class = null
Status = complete
Build = 20230913
Display_Name = Tailscale
Version = 1.48.2-1
Author = Tailscale Inc.
QPKG_File = Tailscale.qpkg
Date = 2023-09-20
Shell = /share/CACHEDEV1_DATA/.qpkg/Tailscale/Tailscale.sh
Service_Port = 41641
Volume_Select = 1
Install_Path = /share/CACHEDEV1_DATA/.qpkg/Tailscale
WebUI = /cgi-bin/qpkg/Tailscale/index.cgi
RC_Number = 101
Use_Proxy = 1
Visible = 1
Enable = TRUE
FW_Ver_Min = 5.0.0
```

QPKG_SERVICE_PORT is converted to Service_Port, so we can use this config to replace the hard code port in Tailscale.sh